conf/modules.config

```diff
@@ -503,16 +503,55 @@ if (!params.skip_plot_fingerprint) {
     }
 }
 
-if (params.macs_gsize) {
+process {
+    withName: 'MACS2_CALLPEAK' {
+        ext.args   = [
+            '--keep-dup all',
+            params.narrow_peak      ? '' : "--broad --broad-cutoff ${params.broad_cutoff}",
+            params.save_macs_pileup ? '--bdg --SPMR' : '',
+            params.macs_fdr         ? "--qvalue ${params.macs_fdr}" : '',
+            params.macs_pvalue      ? "--pvalue ${params.macs_pvalue}" : ''
+        ].join(' ').trim()
+        publishDir = [
+            path: { [
+                "${params.outdir}/${params.aligner}/mergedLibrary/macs2",
+                params.narrow_peak? '/narrowPeak' : '/broadPeak'
+                ].join('') },
+            mode: params.publish_dir_mode,
+            saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
+        ]
+    }
+
+    withName: 'FRIP_SCORE' {
+        ext.args   = '-bed -c -f 0.20'
+        publishDir = [
+            path: { [
+                "${params.outdir}/${params.aligner}/mergedLibrary/macs2",
+                params.narrow_peak? '/narrowPeak' : '/broadPeak',
+                '/qc'
+                ].join('') },
+            enabled: false
+        ]
+    }
+
+    withName: 'MULTIQC_CUSTOM_PEAKS' {
+        publishDir = [
+            path: { [
+                "${params.outdir}/${params.aligner}/mergedLibrary/macs2",
+                params.narrow_peak? '/narrowPeak' : '/broadPeak',
+                '/qc'
+            ].join('') },
+            mode: params.publish_dir_mode,
+            saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
+        ]
+    }
+}
+
+if (!params.skip_peak_annotation) {
     process {
-        withName: 'MACS2_CALLPEAK' {
-            ext.args   = [
-                '--keep-dup all',
-                params.narrow_peak      ? '' : "--broad --broad-cutoff ${params.broad_cutoff}",
-                params.save_macs_pileup ? '--bdg --SPMR' : '',
-                params.macs_fdr         ? "--qvalue ${params.macs_fdr}" : '',
-                params.macs_pvalue      ? "--pvalue ${params.macs_pvalue}" : ''
-            ].join(' ').trim()
+        withName: 'HOMER_ANNOTATEPEAKS_MACS2' {
+            ext.args   = '-gid'
+            // ext.prefix = 'peaks'
```

Comment:
Comment needs to be dealt with here.

Reply:
Just this to sort out now?

Reply:
Fix here [9f783fb](https://github.com/nf-core/chipseq/pull/283/commits/9f783fba987971a4002083f0ba0c7341df83e4ee)